src/resolvers/applicationCycleResolver.ts

```diff
@@ -21,10 +22,19 @@ const applicationCycleResolver:any={
                 return newapplicationCycle;
               },
               async deleteApplicationCycle(_parent:any, _args:any){
-                const deleteapplicationCycle= await applicationCycle.findById(_args.id);
-                if(!deleteapplicationCycle) throw new Error("This applicationCycle doesn't exist")
-                const deletedapplicationCycle = await applicationCycle.findByIdAndRemove(_args.id);
-                return deletedapplicationCycle;
+                      const deleteapplicationCycle= await applicationCycle.findById(_args.id);
+                      if(deleteapplicationCycle !=null){
+                          const name=deleteapplicationCycle.name
+                          const users = await  traineEAttributes.findOne({cohort:deleteapplicationCycle.name})
```

Comment:
currently the cycle( previously named cohort) is in Trainee model not in traineeAttributes model you can check updates on develop and update this one